lib/killbill_client/api/net_http_adapter.rb

```diff
@@ -158,9 +158,6 @@ def request(method, relative_uri, options = {})
             headers = request.to_hash
             headers['authorization'] &&= ['Basic [FILTERED]']
             KillBillClient.log :debug, headers.keys.map { |k| "#{k}='#{headers[k].join(',')}'" }.join(', ')
-            if request.body && !request.body.empty? && request['Content-Type'].include?('application/json')
-              KillBillClient.log :debug, "requestBody='#{request.body}'"
```

Comment:
This has actually been quite useful in the past to debug the client or Kaui.

Isn't the issue that `debug` shouldn't be used in production? Or should we maybe log at a `TRACE` level (not sure if it's [fully](https://github.com/killbill/killbill-plugin-framework-ruby/blob/master/lib/killbill/killbill_logger.rb) supported yet though)?